PendingReleaseNotes.md

```diff
@@ -10,3 +10,5 @@
 
 - Previously, only the latest version of helm was tested and the docs stated only version 3.x of helm as a prerequisite. Now rook supports the six most recent minor versions of helm along with their their patch updates. Explicitly, helm versions 3.13 and newer are supported.
 - Add support for specifying the clusterID in the CephBlockPoolRadosNamespace and the CephFilesystemSubVolumeGroup CR.
+- If a mon is being failed over, if the assigned node no longer exists, failover immediately instead of waiting for a
+  long 20 minute timeout since it appears the same as a node drain.
```

Comment:
Why is it same as a node drain?  In node drain scenario, the node still exists in unschedulable state.  Or maybe I'm not reading this statement correctly.